packages/table/src/table.tsx

```diff
@@ -136,12 +136,16 @@ export interface ITableProps extends IProps, IRowHeights, IColumnWidths {
     loadingOptions?: TableLoadingOption[];
 
     /**
-     * The number of columns to freeze to the left side of the table, counting from the leftmost column.
+     * The number of columns to freeze to the left side of the table, counting
+     * from the leftmost column.
+     * @default 0
```

Comment:
is there any reason why we still need to maintain these values since we appear to be using `numFrozenColumnsClamped` everywhere? If there is a place where we still use the raw `numFrozenColumns` can you add something to this comment explaining why we have to keep this value around?

Reply:
Yeah you're right! Will delete the `@default`s.

Reply:
false alarm

Reply:
Sync'd up offline. We're on the same page now. Adding the `@default`s back in.